toolium/behave/env_utils.py

```diff
@@ -64,7 +64,7 @@ def error(self, exc):
               '           - Exception: %s' % exc
         if self.logger is not None:
             self.logger.error(msg)
-        raise Exception('      ERROR - %s' % msg)
+        self.by_console('      ERROR - %s' % msg)
```

Comment:
Remove this `self.by_console` and replace the `self.logger.error(msg)` previous by `self.logger.warn(msg)` is the same

Reply:
But it is an ERROR, not a WARN. We must to log an ERROR without raising the exception.